src/commands/listCountryMemberCount.ts

```diff
@@ -20,5 +22,12 @@ export const cmdListCountryMemberCount: Middleware<BotContext> = async (
     })
     .sort((a, b) => a.localeCompare(b));
 
+  if (locationCount.length == 0)
```

Comment:
Nice catch! Consider using javascript's strict equality operator here instead:

```suggestion
  if (locationCount.length === 0)
```